stories/Molecules/Sidebar-data/sidebardata.scss

```diff
@@ -24,14 +25,15 @@
 
   &--description {
     a {
+      @extend %img_none;
+
       border: none;
       display: block;
-      font-family: $Font-Family-ProximaRegular;
       font-size: $font-size-14;
+      // font-family: $Font-Family-ProximaNova;
```

Comment:
@harishkumar-srijan Remove this commented code

Reply:
Done